### PR TITLE
CI/CD pipeline based on GH Actions

### PR DESCRIPTION
**Pull request by schaermu**
In order to migrate away from travis due to licensing woes, the current pipeline has to be implemented in GH Actions.

The publishing needs review (e.g. when do we publish, under what conditions).

### CODE REVIEW

**Comment by schaermu** [comment]
Travis will be deactivated once the review was done

**Comment by schaermu** [comment]
@lbovet @Mattiamato any comments/thoughts on this?

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Publish
+on:
+  push:
+    branches:
+      - develop
```
Comment (lbovet) [inline .github/workflows/publish.yml:5]: We publish on pushes to master, not develop.

**Comment by lbovet** [comment]
Weird that we have all past "merges from develop" here. Is it correct?
Nevertheless, I let you merge and work on the branches to make your tries. Please try not to publish try versions on npmjs.com ;)

**Pull request merged**